app/routes/endemics/timings.js

```diff
@@ -0,0 +1,42 @@
+const Joi = require('joi')
+const urlPrefix = require('../../config/index').urlPrefix
```

Comment:
I think Config should be imported here to cover your offer rejection scenario.
const config = require('../../config/index')